### ISSUE TITLE
Directus 8: Error during installation -  Invalid database connection

### ISSUE THREAD

**Issue by andersonbraz** [directus/directus#2496]
# Bug Report

Invalid database connection

## Steps to Reproduce

In a beginner test classroom we are just the installation.

IMPORTANT: We are following the https://docs.directus.io/getting-started/installation.html documentation.

## Step - 1 (Good)
![step_error_directus_001](https://user-images.githubusercontent.com/1479033/70547998-9cc96780-1b50-11ea-9503-6da51c5378fa.png)

## Step - 2 (Good)
![step_error_directus_002](https://user-images.githubusercontent.com/1479033/70548041-ad79dd80-1b50-11ea-960b-4f50e4cfd34d.png)

## Step - 3 (Good)
![step_error_directus_003](https://user-images.githubusercontent.com/1479033/70548099-c1bdda80-1b50-11ea-81b3-096a072c141a.png)

## Step - 4 (Ops! Error)
![step_error_directus_004](https://user-images.githubusercontent.com/1479033/70548117-c71b2500-1b50-11ea-9006-be8186f6d94f.png)

## Technical Details

- Device: [MacBook Air]
- OS: [MacOS Catalina 10.15.1]
- Web Server: [ Apache 2.4.41]
- PHP Version: [7.3.12]
- Database: [MySQL 8.0.18]
- Directus Release Version: [v8.2.0]
- Install Method: [download pacakge zip on official repository]

**Comment by rijkvanzanten** [comment]
Are your database credentials correct? Can you connect to MySQL with those exact credentials through other means?

**Comment by andersonbraz** [comment]
Yeah! my database credentials are correct and yes, i tested credentials through other means

**Comment by rijkvanzanten** [comment]
Strange! What do you use to run Apache on your Mac?

**Comment by andersonbraz** [comment]
$ sudo apachectl start

But my Apache start on auto when open macOS .

More Apache details:
$ apachectl -V
Server version: Apache/2.4.41 (Unix)
Server built:   Oct  1 2019 10:31:38
Server's Module Magic Number: 20120211:88
Server loaded:  APR 1.7.0, APR-UTIL 1.6.1
Compiled using: APR 1.7.0, APR-UTIL 1.6.1
Architecture:   64-bit
Server MPM:     prefork
  threaded:     no
    forked:     yes (variable process count)
Server compiled with....
 -D APR_HAS_SENDFILE
 -D APR_HAS_MMAP
 -D APR_HAVE_IPV6 (IPv4-mapped addresses enabled)
 -D APR_USE_SYSVSEM_SERIALIZE
 -D APR_USE_PTHREAD_SERIALIZE
 -D SINGLE_LISTEN_UNSERIALIZED_ACCEPT
 -D APR_HAS_OTHER_CHILD
 -D AP_HAVE_RELIABLE_PIPED_LOGS
 -D DYNAMIC_MODULE_LIMIT=256
 -D HTTPD_ROOT="/usr/local/Cellar/httpd/2.4.41_1"
 -D SUEXEC_BIN="/usr/local/opt/httpd/bin/suexec"
 -D DEFAULT_PIDLOG="/usr/local/var/run/httpd/httpd.pid"
 -D DEFAULT_SCOREBOARD="logs/apache_runtime_status"
 -D DEFAULT_ERRORLOG="logs/error_log"
 -D AP_TYPES_CONFIG_FILE="/usr/local/etc/httpd/mime.types"
 -D SERVER_CONFIG_FILE="/usr/local/etc/httpd/httpd.conf"

**Comment by rijkvanzanten** [comment]
Can you reach your MySQL installation from this local Apache server?

**Comment by andersonbraz** [comment]
Uau! Excellent idea.

I tested like this:

File: test.php
Code:

`<?php

$link = mysqli_connect("localhost", "root", "0okm9ijn", "sample");

if (!$link) {
    echo "Error: Unable to connect to MySQL." . PHP_EOL;
    echo "Debugging errno: " . mysqli_connect_errno() . PHP_EOL;
    echo "Debugging error: " . mysqli_connect_error() . PHP_EOL;
    exit;
}

echo "Success: A proper connection to MySQL was made! The my_db database is great." . PHP_EOL;
echo "Host information: " . mysqli_get_host_info($link) . PHP_EOL;

mysqli_close($link);

?>`

Result in browser:

Warning: mysqli_connect(): The server requested authentication method unknown to the client [caching_sha2_password] in /Users/andersonbraz/Sites/directus/public/test.php on line 3

Warning: mysqli_connect(): (HY000/2054): The server requested authentication method unknown to the client in /Users/andersonbraz/Sites/directus/public/test.php on line 3
Error: Unable to connect to MySQL. Debugging errno: 2054 Debugging error: The server requested authentication method unknown to the client


---

I solved like this:

ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password
BY 'here_my_password';

Result in browser:

Success: A proper connection to MySQL was made! The my_db database is great. Host information: Localhost via UNIX socket

Thank's  @rijkvanzanten
